CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_shopify_source v0.16.0  
+[PR #97](https://github.com/fivetran/dbt_shopify_source/pull/97) includes the following updates:  
+
+## Breaking Changes  
+- Introduced the `inventory_quantity` source and the `stg_shopify__inventory_quantity` model to support downstream inventory quantity tracking. See the [documentation](https://fivetran.github.io/dbt_shopify_source/#!/model/model.shopify_source.stg_shopify__inventory_quantity) for details on the newly added columns.  
```

Comment:
I'd maybe add a comment about how `quantity` was deprecated from `inventory_level`, just so it's easy to track why this change was needed